helm/event-exporter-app/templates/deployment.yaml

```diff
@@ -15,29 +18,24 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
     spec:
-      serviceAccountName: event-exporter-app
-      securityContext:
-        {{- with .Values.podSecurityContext }}
-          {{- . | toYaml | nindent 8 }}
-        {{- end }}
+      serviceAccountName: {{ include "event-exporter.fullname" . }}
+      {{- if .Values.podSecurityContext }}
+      securityContext: {{ toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- end }}
       containers:
-        - name: event-exporter-app
-          image: "{{ .Values.registry.domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
-          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
-          securityContext:
-            {{- with .Values.securityContext }}
-              {{- . | toYaml | nindent 12 }}
-            {{- end }}
-          args:
-            - -conf=/data/config.yaml
-          volumeMounts:
-            - mountPath: /data
-              name: cfg
+      - name: event-exporter
+        image: {{ .Values.registry.domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        {{- if .Values.securityContext }}
+        securityContext: {{ toYaml .Values.securityContext | nindent 10 }}
+        {{- end }}
+        args:
+        - --conf
+        - /etc/event-exporter/config.yaml
+        volumeMounts:
+        - name: config
+          mountPath: /etc/event-exporter
```

Comment:
Why the change of the mount path? Is this directory unused within the container image?

Reply:
No special reason, just thought it would be better to place config in a directory which actually is about configuration. 😬

Reply:
I'd rather we didn't change the path, just in case something expected it to be there.

Reply:
Sure, gonna revert that!

Reply:
Done.